src/methods.js

```diff
@@ -26,6 +26,47 @@ const logMessage = ({ type, msg, ignoreConsole }) => {
 	return { logColor, logMsg }; // Return for testing purposes
 };
 
+const processTestSuiteResults = (suiteResults, sort) => {
+	if (!sort || sort === 'default') {
+		return suiteResults;
+	}
+	if (sort === 'status') {
```

Comment:
I would suggest splitting out the code block within the if-statement into a method named `sortByStatus` within a new file called `sorting.js` that would then be required and used here. This would improve the readability and also allow for expanding with additional sorting methods in the future

Reply:
good idea, done